### DISCUSSION TITLE
wxmaxima flickering screen problem

### DISCUSSION THREAD

**Discussion by mamboze** [wxMaxima-developers/wxmaxima#1548] [Q&A]
I have been using wxmaxima for a few years without any problems at all. But when I returned to it recently I found that when it was run, a message stating that  the GTK_IM_MODULE is set to "xim" and that "horrible" flicking would occur. This was exactly true. I now find that due to the screen flickering in the active area of the page wxmaxima is nearly unusable. The rest of the page is stable, no problem at all.

I checked out the error online, but could not find a work-around or any other solution. I was also unable to find out where GTK_IM_MODULE  is located on my system and how it might be reset. My use of wxmaxima prior to this was entirely without any screen problems at all. So this is really a big hassle.

I run wxmaxima on an HP Pavilion laptop with Linux Mint 19.3 installed. Any help would be very welcome.

**Comment by gunterkoenigsmann** [comment]
In my case installing ibis and ibus3 and ibus-gtk somehow made my gnome applications work again.

**Comment by mamboze** [comment]
Using synaptic selection for "ibus" and installing the various files, including  ibus-gtk3, did not fix the flicker. I've attached a brief video showing the problem.

If you have any other suggestions, I'm certainly happy to try them; 

Thank you for you help. I really appreciated it. 

https://user-images.githubusercontent.com/565749/119251725-5b653480-bbeb-11eb-8f91-dacb56141c31.mp4

**Comment by kunnutvaan** [comment]
I was affected by this for many years. And now I do hope, I solved this for me!

Steps I did on my Xubuntu 22.04 laptop:
I made sure ibus and ibus-gtk-3 and ibus-gtk-4 were installed.
Then I made sure .xinputrc is not present in ~ .
After these rebooted the machine. And as of now no flickering.

Also

env | grep -i gtk

shows the magical result:

GTK_IM_MODULE=ibus

instead of xim.
